### PR TITLE
ADD - access control for projects using the applier

### PR DESCRIPTION
**Pull request by springdo**
Simple template and params to add role binding to each project created so when running the applier again; these are not lost.

### CODE REVIEW

File: openshift-templates/project-acl/template.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (sherl0cks) [inline openshift-templates/project-acl/template.yml]: Can you merge this template with https://github.com/rht-labs/labs-ci-cd/blob/master/openshift-templates/rolebinding-group/template.yaml

File: params/project-acl/all [hunk 2]
```diff
@@ -0,0 +1,2 @@
+GROUP_NAME=c10-companyxyz-ocp-devs
```
Comment (sherl0cks) [inline params/project-acl/all]: per our discussion, please use the group name `labs-ci-cd-contributors`

File: inventory/group_vars/all.yml [hunk 3]
```diff
@@ -20,6 +20,20 @@ openshift_cluster_content:
     params: "{{ inventory_dir }}/../params/projectrequests/demo"
     tags:
     - projects
+- object: project-acls
```
Comment (sherl0cks) [inline inventory/group_vars/all.yml]: can we include an acl for the demo project too?

**Comment by springdo** [comment]
@sherl0cks  - realise i've not looked at this one in a while; but when we reshuffle the inventory around as in #159 and #164  there'll be more rework here so will do this once they are  closed.

**Comment by makentenza** [comment]
LGTM

**Pull request merged**